class.jetpack-jitm.php

```diff
@@ -414,6 +426,62 @@ function videopress_media_upload_warning_msg() {
 		<?php
 	}
 
+	/**
+	 * Display message prompting user to install WooCommerce Services.
+	 *
+	 * @since 4.6
+	 */
+	function woocommerce_services_msg() {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
+			return;
+		}
+
+		if ( isset( self::$jetpack_hide_jitm['woocommerce_services'] ) ) {
+			return;
+		}
+
+		if ( ! function_exists( 'wc_get_base_location' ) ) {
+			return;
+		}
+
+		$base_location   = wc_get_base_location();
+		$store_in_usa    = ( 'US' == $base_location['country'] );
+		$store_in_canada = ( 'CA' == $base_location['country'] );
+
+		if ( $store_in_usa ) {
+			$message = __( 'Try our new service for USPS shipping & label-printing.', 'jetpack' );
+		}
+		elseif ( $store_in_canada ) {
+			$message = __( 'Try our new Canada Post shipping service.', 'jetpack' );
+		}
+		else {
+			return;
+		}
+
+		$install_url = wp_nonce_url( add_query_arg( array( 'wc-services-action' => 'install' ) ), 'wc-services-install' );
```

Comment:
Hmm.. okay - after a quick discussion, I think we'll take the user to the plugins page upon installation.